src/localMetaConfiguration.h

```diff
@@ -21,26 +21,29 @@
 
 #include "metaIOConfig.h"
 
-#if defined(METAIO_FOR_ITK) || !defined(METAIO_FOR_VTK)
+#if defined(METAIO_FOR_ITK)
 // ITK
 
 #  define METAIO_USE_NAMESPACE 0
 #  define METAIO_NAMESPACE ITKMetaIO
+#  define METAIO_STREAM std
 
-#  include "itk_zlib.h"
+#  include <itk_zlib.h>
 
 #  include <iostream>
 #  include <fstream>
 
 #  define METAIO_EXPORT
 
-#else
+#elif defined(METAIO_FOR_VTK)
 // VTK
 
 #  define METAIO_USE_NAMESPACE 1
 #  define METAIO_NAMESPACE vtkmetaio
+#  define METAIO_STREAM vtksys
 
-#  include "vtk_zlib.h"
+#  include <vtksys/FStream.hxx>
+#  include <vtk_zlib.h>
 
 #  include <iostream>
 #  include <fstream>
```

Comment:
Why include metaIO exports for VTK?

Reply:
Yeah - we probably should just remove all reference to VTK in a separate pull request.   Perhaps MetaIO should also just become part of ITK instead of existing as a separate repo.   I'll chat with the ITK folks about that.   Probably a topic for another pull request. :)